### PR TITLE
Why code?

### PR DESCRIPTION
**Pull request by lakikowolfe**
If your pull request is for adding a blog post, please check that you include the following:

- [x] _Title_ - A short, catchy description to entice readers
- [x] _Author_ - Your name, which should match your contributor [file](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#adding-yourself-as-a-contributor). If you do not have a contributor file, please feel free to contact one of our team members to assist you
- [x] _Categories_ - Select 1 at minimum of either Community, Technical, or Science as [describe here](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)
- [x] _Tags_ - Suggested keywords that can help readers find information from among our blog posts. Please suggest a few tags following [our guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#tags).
- [x] _Content_ - This is the body of the blog post! We recommend relatively short posts (no more than five paragraphs), equivalent to a three minute read. Pictures and links to other sites are also great!

The Coop team member reviewing the submission will check for the following:

- [x] The post is appropriate for the blog; [see guidelines](https://github.com/FredHutch/coop/blob/gh-pages/CONTRIBUTING.md#what-kind-of-posts-are-appropriate-for-our-blog)

- [x] The author is added as a [contributor](https://fredhutch.github.io/coop/contributors/) to the blog

- [x] The file is in the DRAFT folder

- [x] A plan for publication (adding `YEAR-MONTH-DAY` to filename and moving to posts folder).

If the pull request is not for adding a blog then delete the content above and briefly describe below the changes proposed in the pull request.

### CODE REVIEW

**Comment by lakikowolfe** [comment]
This is a pretty rough dump of my thoughts - I would definitely appreciate some thoughts. Right now the scope is basically it's important to understand downstream methods and the downfalls of excel.

File: _drafts/why-code.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: "Why learn to code?"
+author: Lauren Wolfe
+categories: # delete the categories that do not apply and keep at least one
+  - Community
+tags: # add 1-4 lowercase tags that are relevant to your post, ex: r, python, genomics, workflows
+  - r
+  - python
+  - 
+---
+Occassionally, after someone takes an Intro R or Python course through FredHutch.io we will get feedback that boils down to the question "How am I supposed to use this in my research?". It's an understandable question - an intro course generally covers the basics of importing, exporting, subsetting, and visualizing data. This basic skillset does not clearly translate to the incredible and complex work being done in the clinics and wet-labs around campus. In this post we will go over how learning to program and understanding code could improve biology research.
```
Comment (k8hertweck) [inline _drafts/why-code.md]: Let's reframe this: "so you're thinking about taking a coding course, but aren't sure if it will apply to your own research."

**Comment by k8hertweck** [comment]
This is a great start on a post that is difficult to write!

File: _drafts/why-code.md (hunk 1 above)
Comment (lakikowolfe) [reply _drafts/why-code.md]: Ok reworked it a bit - let me know what you think

File: _drafts/why-code.md [hunk 2]
```diff
@@ -8,14 +8,18 @@ tags: # add 1-4 lowercase tags that are relevant to your post, ex: r, python, ge
...
+Many articles on what computational biology brings to the table focus on complex, compute intensive methods. However, even having a basic knowledge of coding can improve day-to-day life in the laboratory. Excel has long been used as a universal database, analysis engine, and data visulaization platform by biologists. While this tool may be commonly used it's downfalls are well documented. A [study in 2016](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1044-7) found that Microsoft Excel was causing widespread errors in scientific literature by converting gene symboles to dates. Further, the point-and-click nature of the software makes reproducible analyses difficult and introducing accidental changes into the data easy.
```
Comment (k8hertweck) [inline _drafts/why-code.md]: visulaization -> visualization

**Comment by k8hertweck** [comment]
- add link to beginners page from wiki
- add link to [fredhutch.io courses](http://www.fredhutch.io/software/), explain they're focused on skills relevant to researchers

File: _drafts/why-code.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Small scale improvements
+
+Many articles on what computational biology brings to the table focus on complex, compute intensive methods. However, even having a basic knowledge of coding can improve day-to-day life in the laboratory. Excel has long been used as a universal database, analysis engine, and data visulaization platform by biologists. While this tool may be commonly used it's downfalls are well documented. A [study in 2016](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1044-7) found that Microsoft Excel was causing widespread errors in scientific literature by converting gene symboles to dates. Further, the point-and-click nature of the software makes reproducible analyses difficult and introducing accidental changes into the data easy.
```
Comment (k8hertweck) [inline _drafts/why-code.md]: it's -> its

**Comment by lakikowolfe** [comment]
@k8hertweck this could use another look through!

**Comment by lakikowolfe** [comment]
Edits to make from a conversation with Kate: 
- More provocative intro
- change subheadings to be more encompassing of the main point

File: _drafts/why-code.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+* **[The Fred Hutch Biomedical Data Science Wiki](https://sciwiki.fredhutch.org/):** Hutch specific written documentaiton of policies, tools, and resources available to researchers supporting the generation, analysis, and sharing or research data. See [this](https://sciwiki.fredhutch.org/scicomputing/software_overview/) page for an overview of how to get started with software development.
+
+* **[The Bioinformatics and Data Science Cooperative (The Coop)](https://research.fhcrc.org/coop/en.html):**: If you're here on this blog hopefully you know about The Coop. But if not - here at The Coop we are dedicated to acting as a single source of information about resources and activities related to data-intensive science at the Hutch. Any questions you might have about training, tools, best practice, or resources can be routed to either Kate Hertwick or me (Lauren Wolfe) by Slack, Teams, or email. Stay up to date with the latest and greatest data-intensive research related offerings by emailing `coophelp@fredhutch.org` and signing up for our [newsletter](http://research.fhcrc.org/coop/en/newsletter.html).
```
Comment (k8hertweck) [inline _drafts/why-code.md]: remove extra :

File: _drafts/why-code.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  - Community
+tags: # add 1-4 lowercase tags that are relevant to your post, ex: r, python, genomics, workflows
+  - r
+  - python
+  - training
+  - coding
+---
+Chances are, that if you're a biologist in the 21st century, you have been told at some point that you should learn to code. In fact, when I graduated from Western Washington University in 2014 it was said to me! So, shortly after graduation I took my first coding class. It was in C++ and I made a `hello world` script and some calculators. I walked away excited that I had learned something new but with no idea how this would one day apply to the biology work I hoped to do. Software and coding in biology tends to be associated with cutting-edge projects that require in-depth computational and statistical knowledge and access to high-powered compute environments. When all we hear about are the cutting-edge, paradigm-shifting ways that code is used to make biological insights it leaves beginners wondering when, if ever, the skillset will become useful to them. However, even a more basic knowledge of coding can improve research and make mundane lab tasks quicker and more effective.
```
Comment (k8hertweck) [inline _drafts/why-code.md]: One of the most common questions we receive at The Coop is whether someone in our community should learn to code. Changes are good that you've heard someone talk about interesting research they've accomplished with coding, or someone has told you that you should learn to code.

**Comment by k8hertweck** [comment]
Added a comment with a suggestion for the first sentence, and a typo correction. Feel free to publish when you're ready!

**Pull request merged**